src/inet.c

```diff
@@ -49,6 +49,9 @@ int uv_inet_ntop(int af, const void* src, char* dst, size_t size) {
   /* NOTREACHED */
 }
 
+char* uv_if_indextoname(unsigned int ifindex, char* ifname) {
+    return if_indextoname(ifindex, ifname);
```

Comment:
Also, I would probably change the signature of the function so it returns an `int` so it returns 0 on success or `-errno` on error.

On Windows apparently if `if_indextoname` returns NULL there's no way to check the error, so it's probably better use this other method explained in the [documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/bb408408(v=vs.85).aspx):

> The if_indextoname function is implemented for portability of applications with Unix environments, but the ConvertInterface functions are preferred. The if_indextoname function can be replaced by a call to the ConvertInterfaceIndexToLuid function to convert an interface index to a NET_LUID followed by a call to the ConvertInterfaceLuidToNameA to convert the NET_LUID to the ANSI interface name.

> If the if_indextoname fails and returns a NULL pointer, it is not possible to determine an error code.